drivers/encx24j600/encx24j600.c

```diff
@@ -360,7 +360,7 @@ static int _recv(netdev_t *netdev, void *buf, size_t len, void *info)
     /* hdr.frame_len given by device contains 4 bytes checksum */
     size_t payload_len = hdr.frame_len - 4;
 
-    if (buf) {
+    if (buf && (len >= payload_len)) {
```

Comment:
Yeah, considered something like that, but was counting on your suggestion 😉 Will adapt, also note: the current docu of recv in [1], does not cover the case of `buffer to small` is just covers `no buffer`.

[1]: https://github.com/RIOT-OS/RIOT/blob/master/drivers/include/net/netdev.h#L322